app/src/app/components/view-profile/view-profile.component.ts

```diff
@@ -53,18 +52,19 @@ export class ViewProfileComponent implements OnInit {
   }
 
   populateForm(userInfo: Person) {
-    this.profileForm = this.formBuilder.group({
-      jobTitle: [userInfo?.jobTitle || ''],
-      location: [userInfo?.location || ''],
-      telephone: [userInfo?.telephone || '', Validators.pattern('^[0-9]*$')],
-      mobile: [userInfo?.mobile || '', Validators.pattern('^[0-9]*$')],
-      oldPassword: [''],
-      newPassword: [''],
-      verifyPassword: [''],
-      companyPostCode: [userInfo?.company?.postcode || ''],
-      companyAddress: [userInfo?.company?.address1 || ''],
-      companyTelephone: [userInfo?.company?.telephone || '', Validators.pattern('^[0-9]*$')],
-      companyEmail: [userInfo?.company?.email || '', Validators.email]
+    this.profileForm = new FormGroup({
```

Comment:
I have put down separate forms for all the input fields because I need to check the error for individual input field on the UI also, so that error message could be displayed only for the particular field.